package.json

```diff
@@ -3,13 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "ajv": "^6.5.4",
```

Comment:
Why do you need this package?

Reply:
Because I had problems with adding `react-router-dom` to package.json and after some research based on errors in my terminal I saw that manual installation of this package can solve it - and indeed, it has.

Reply:
Remove when it will work without this, please